src/stores/SetupEncryptionStore.js

```diff
@@ -70,7 +76,7 @@ export class SetupEncryptionStore extends EventEmitter {
 
     async fetchKeyInfo() {
         const keys = await MatrixClientPeg.get().isSecretStored('m.cross_signing.master', false);
-        if (!keys || Object.keys(keys).length === 0) {
+        if (Object.keys(keys).length === 0) {
```

Comment:
this needs that null-guard as `isSecretStored` can return `null`

Reply:
Bleh, well spotted. I actually did this and then somehow forgot to commit it...